### DISCUSSION TITLE
Copilot + Vim: Trigger Inline Suggestion

### DISCUSSION THREAD

**Discussion by lowmelvin** [community/community#31034] [Copilot Conversations]
For Vim, is there a way to have Copilot make an inline suggestion only on command? This would be a great feature as I generally don't want Copilot turned on all the time.

I noticed that this feature exists in VSCode and also in Jetbrains, but not yet for Vim--unless I'm missing something.

Thanks!

**Comment by samodostal** [comment]
I was missing the exact same feature. That's why I made this plugin (only available for neovim): https://github.com/samodostal/copilot-client.lua.
